src/app/components/harvest/components/metadata-review/folder-row.component.ts

```diff
@@ -146,18 +150,38 @@ export class FolderRowComponent {
     return this.harvestItem.report;
   }
 
+  public get hasHarvestItems(): boolean {
+    return !this.mapping && !(this.row.isRoot && !+this.row.harvestItem);
+  }
+
   public constructor(private injector: Injector) {}
 
+  public ngOnChanges(): void {
+    // to inherit mappings values among ancestor items, if the parent item gains a Harvest Mapping, then all the child items should too
```

Comment:
_This method is currently faulty_  
Since folder rows only have knowledge of their parents, it is not possible to make rows inherit their parents mappings before they are shown on the DOM.

Therefore, I either need to change this.rows to be a doubly linked list (with knowledge of the children, so that I can recurse down the tree when a mapping is changed on a parent item), or move this logic to where the mappings change request is sent to the api (where it does have knowledge of the full model).